components/JEDI/JEDICard.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import styles from '../../styles/pages/jedi.module.scss';
+
+// TODO: flesh out docs
+/**
+ * creates a JEDI card from props; for now, uses <img>
+ * for image instead of next/image
+ */
+function JEDICard({ name, pronouns, bio, pic, alt }) {
```

Comment:
Similarly here, can we re-use any of the `Officer` component that we use in `/about`?

Reply:
Hm, so I'd like to eventually nuke the `/about` officer component once we get #111 up and running. I do see quite a bit of overlap between this and the same component in #260, though there are slight style differences. Overall, what do you think? I'm also down to punt this concern until later, and then have a PR to unify everything once stuff is in.

Reply:
I think it's valuable to invest in a generic `OfficerCard` component for a more unified design. Not a huge blocker if not in-scope for this PR. If so let's just create a new issue and tackle it after all the high priority stuff is done.

Reply:
Sounds good, if you don't mind I'll make a new issue then - this'll also factor into the Impact page + the dev team officers page, so I'll find some time to fit that in.

Reply:
Created #264.